docker/easy-setup.sh

```diff
@@ -62,12 +62,31 @@ function Help(){
     echo -e "       Defines the path where Elasticsearch will store it's data. The path must already exists and the current user must have write permissions. Default will be in a named docker volume ('/var/lib/docker')"
     echo -e "       The interactive prompt regarding this option will be skipped\n"
     echo -e " --es-memory"
-    echo -e "       Amount of memory to give to the elasticsearch java heap. Default is '2G'. Accepted units are 'm','M','g','G'. ex \"--es-memory 512m\" or \"--es-memory 4G\". Default is '2G'\n"
+    echo -e "       Amount of memory to give to the elasticsearch java heap. Accepted units are 'm','M','g','G'. ex \"--es-memory 512m\" or \"--es-memory 4G\". Default is '2G'\n"
+    echo -e " --ls-memory"
+    echo -e "       Amount of memory to give to the logstash java heap. Accepted units are 'm','M','g','G'. ex \"--es-memory 512m\" or \"--es-memory 4G\". Default is '2G'\n"
+    echo -e " --restart-mode"
+    echo -e "       'no', 'always', 'on-failure','unless-stopped'"
```

Comment:
Can you add description of the options please?

Reply:
done.